Application.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        Lottery ball = new Lottery();
+        for(int i = 0; i < 3; i++){
+            System.out.println(ball.getRandomBall().toString());

```

Comment:
There is no need to call toString(). When we pass some object to the println() method, toString() is called implicitly inside the method anyway.